src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -76,56 +90,37 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       response.setContentType("application/json;");
       response.getWriter().println(gson.toJson(events));
     } else {
-      // ToDo: Seokha Kang implement this method.
-      response.getWriter().println("You've called the get request");
+      Long event_id = Long.parseLong(request.getParameter("event_id"));
+      KeyFactory keyFactory = datastore.newKeyFactory().setKind("Event");
+      Entity entity = datastore.get(keyFactory.newKey(event_id));
+      Event event = createEventFromEntity(entity);
+
+      Gson gson = new Gson();
+      response.setContentType("application/json;");
+      response.getWriter().println(gson.toJson(event));
+      System.out.println(gson.toJson(event));
```

Comment:
nit: remove system out print

Reply:
Done! Also, I copy-pasted this from Seokha's pull request just to see how the entire file would be laid out, wouldn't want to take credit for it myself.

Reply:
It is okay for now, but for future reference, wait a bit for the PR owner to submit their code (maybe unless it's an emergency).